### PR TITLE
Migration adding Kambule Doctoral Dissertation Award

### LINKED ISSUE #611: Configure Kambule Doctoral Dissertation Award 2020
Create a migration that inserts the following new records into our database:
- Add an [event](https://github.com/deep-learning-indaba/Baobab/blob/ed9ef8986eeb5d725ce24cffdbbec15b7224c83a/api/app/events/models.py#L12) with the following data:
  - name: Thamsanqa Kambule Doctoral Dissertation Award 2020
  - description: Thamsanqa Kambule Doctoral Dissertation Award 2020
  - start_date: 23 August 2020
  - end_date: 28 August 2020
  - key: kambuledoctoral2020
  - organisation_id: 1
  - email_from: baobab@deeplearningindaba.com
  - url: http://www.deeplearningindaba.com
  - application_open: 1 March 2020
  - application_close: 17 April 2020
  - review_open: 25 April 2020
  - review_close: 15 May 2020
  - selection_open: 1 Jan 2020
  - selection_close: 1 Jan 2020
  - offer_open: 1 Jan 2020
  - offer_close: 1 Jan 2020
  - registration_open: 1 Jan 2020
  - registration_close: 1 Jan 2020
  - event_type: AWARD

- Add an [application_form](https://github.com/deep-learning-indaba/Baobab/blob/ed9ef8986eeb5d725ce24cffdbbec15b7224c83a/api/app/applicationModel/models.py#L4) with the following data
  - event_id: The one created above!
  - nominations: True

- Configure the [Sections](https://github.com/deep-learning-indaba/Baobab/blob/ed9ef8986eeb5d725ce24cffdbbec15b7224c83a/api/app/applicationModel/models.py#L20) and [Questions](https://github.com/deep-learning-indaba/Baobab/blob/ed9ef8986eeb5d725ce24cffdbbec15b7224c83a/api/app/applicationModel/models.py#L35) according to [this document](https://docs.google.com/document/d/19f8DvO806ZJ2UQxgvjs-lJo7B8waAd-_qixk3hv57Zk/edit#heading=h.n8mt72nzmyr9)

Have a look at [previous migrations](https://github.com/deep-learning-indaba/Baobab/blob/develop/api/migrations/versions/87213d612eaf_eeml_application_form.py) for inspiration (although note that the data model may have changed since then!)

### PR DESCRIPTION
**Pull request by OdendaalG**
The changes aren't done yet. This is more to get feedback and help with doing migrations.
Resolves #611

### CODE REVIEW

File: api/migrations/versions/f4bc3108c330_.py [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+        section_id=candidate_information.id,
+        headline='Title',
+        placeholder='Title',
+        order=1,
+        questionType='short-text',
+        validation_regex=1,
+        is_required=True
+    )
+    candidate_q1.depends_on_question_id=q1_nomination_capacity.id
+    candidate_q1.show_for_values='other'
```
Comment (avishkar58) [inline api/migrations/versions/f4bc3108c330_.py]: We want this field to be an array - ie `['other']` - same for questions below

**Comment by OdendaalG** [comment]
Hi @avishkar58, moving it to a later migration seemed to work! Let me know if there are any other changes I can implement.

File: api/migrations/versions/14d53fa0c9c8_.py [hunk 2]
```diff
@@ -0,0 +1,556 @@
...
+        options=[
+            {'label':'Self-nomination', 'value':'self'},
+            {'label':'Nomination on behalf of a candidate','value':'other'}
+        ]
+    )
+    session.add(q1_nomination_capacity)
+    session.commit()
+
+    nominator_information = Section(app_form_id, 'Nomination Information', """
+Details of the person nominating the doctoral candidate. Dependency: Only if 'Nominating Capacity' question has answer 'Nomination on behalf a candidate'
```
Comment (avishkar58) [inline api/migrations/versions/14d53fa0c9c8_.py]: Can you fill in the dependency fields here and remove the Dependency part of the description :)

Comment (OdendaalG) [reply api/migrations/versions/14d53fa0c9c8_.py]: No problem! Seems I overlooked it, will change it promptly.

**Comment by OdendaalG** [comment]
Hi @avishkar58, I wouldn't mind tackling the other awards. Its issue 609 and 610 correct?

**Comment by avishkar58** [comment]
> Hi @avishkar58, I wouldn't mind tackling the other awards. Its issue 609 and 610 correct?

Correct - thanks! Much appreciated! :)

**Pull request merged**